src/components/CompareResults/ResultsView.tsx

```diff
@@ -50,6 +52,17 @@ function ResultsView(props: ResultsViewProps) {
           isBaseSearch={null}
           expandBaseComponent={() => null}
         />
+
+        {intervalValue ?? (
+          <CompareOverTime
```

Comment:
Yes, that makes sense! It got pretty knarly trying to fit CompareOverTime into the ResultsView with CompareWithBase. This will make it a lot simpler.